dask/utils.py

```diff
@@ -227,3 +227,20 @@ def getargspec(func):
         return inspect.getargspec(func.func)
     else:
         return inspect.getargspec(func)
+
+
+def is_integer(i):
+    """
+    >>> is_integer(6)
+    True
+    >>> is_integer(42.0)
+    True
+    >>> is_integer('abc')
+    False
+    """
+    if isinstance(i, int):
```

Comment:
According to SO, that will fail for larger floats. http://stackoverflow.com/questions/21583758/how-to-check-if-a-float-value-is-a-whole-number#comment41541791_21583824

I can use `issubtype(i.dtype, np.integer)` and punt for "integer" numpy floating point numbers.